### PR TITLE
149 update collection metadata

### PR DESCRIPTION
**Pull request by ShanaLMoore**
# Summary
This PR updates the collection metadata because the client has since made updates to their [spreadsheet](https://docs.google.com/spreadsheets/d/1_0QVbQU_wj3ITUih5dGPGkWHN0QyhGO9hKSf6rXwKPc/edit#gid=365080784). 

ref #149 

# Screenshots / Video
Screen Shot 2022-11-03 at 12 43 12 PM

# Expected Behavior

- [ ] The Collection metadata should reflect the client's spreadsheet
- [ ] A user should be able to fill in all of the fields and save a collection
- [ ] A user should be able to create a collection via a bulkrax csv import
  - [ ] resource_link will not be available via the form, but the user should be able to set it via bulkrax.

Updated sample file:

[149-all-collection-metadata.csv](https://github.com/scientist-softserv/utk-hyku/files/9932879/149-all-collection-metadata.csv)

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -96,15 +96,15 @@ def self.modified_field
...
+    config.add_index_field 'resource_link', label: 'Resource Link'
     config.add_index_field 'date_created_d_tesim', label: 'Machine Readable Creation Date'
     config.add_index_field 'date_issued_tesim', label: 'Date Issued'
     config.add_index_field 'date_issued_d_tesim', label: 'Machine Readable Publication Date'
     config.add_index_field 'extent_tesim', label: 'Extent'
     config.add_index_field 'form_tesim', label: 'Form'
     config.add_index_field 'publication_place_tesim', label: 'Publication Place'
     config.add_index_field 'repository_tesim', label: 'Repository'
-    config.add_index_field 'rights_notes_tesim', label: 'Rights Notes'
+    config.add_index_field 'note_tesim', label: 'note'
```
Comment (laritakr) [inline app/controllers/catalog_controller.rb]: Should the label be capitalized?

File: app/controllers/catalog_controller.rb [hunk 2]
```diff
@@ -132,14 +132,14 @@ def self.modified_field
...
-    config.add_show_field 'collection_link', label: 'Collection Link'
+    config.add_show_field 'resource_link', label: 'Resource Link'
     config.add_show_field 'date_created_d_tesim', label: 'Machine Readable Creation Date'
     config.add_show_field 'date_issued_tesim', label: 'Date Issued'
     config.add_show_field 'date_issued_d_tesim', label: 'Machine Readable Publication Date'
     config.add_show_field 'form_tesim', label: 'Form'
     config.add_show_field 'publication_place_tesim', label: 'Publication Place'
     config.add_show_field 'repository_tesim', label: 'Repository'
-    config.add_show_field 'rights_notes_tesim', label: 'Rights Notes'
+    config.add_show_field 'note_tesim', label: 'note'
```
Comment (laritakr) [inline app/controllers/catalog_controller.rb]: Label is lower case here too...

File: .github/workflows/build-test-lint.yaml [hunk 3]
```diff
@@ -9,4 +9,4 @@ on:
 
 jobs:
   call-workflow:
```
Comment (ShanaLMoore) [inline .github/workflows/build-test-lint.yaml:11]: This change was [requested](https://assaydepot.slack.com/archives/G031E2P1RJM/p1667509131464649) by LeaAnn/Team Sprint. They deleted a branch that we were pointing to. Not updating this causes the pipeline's build to fail.

**Review by laritakr — APPROVED** [review]
Looks good!

File: app/controllers/catalog_controller.rb (hunk 1 above)
Comment (ShanaLMoore) [reply app/controllers/catalog_controller.rb]: Yes, I'll update that. Thanks!

**Pull request merged**